e2e/pages/disk-list.po.ts

```diff
@@ -40,4 +40,21 @@ export class DiskList extends CloudstackUiPage {
   clickClose() {
     element(by.css('.backdrop.ng-star-inserted')).click();
   }
+
+  clickOnReadyDisk() {
+    element
+      .all(by.css('.entity-card.mat-card'))
+      .first()
```

Comment:
In this method, you click on the first available disk, but you must click on the disk in the status "Ready"